app/templates/introduction.html

```diff
@@ -59,8 +59,8 @@ <h2 class="saturn">{{ _("You are completing this for <span>%(ru_name)s</span>%(t
             {% endif %}
 
             {%- if legal_basis -%}
-                <h2 class="neptune u-mb-xs">{{ _("Your response is legally required") }}</h2>
-                <p class="mars u-mb-m">{{ legal_basis }}</p>
+                <h2 class="neptune u-mb-xs" id="legal-response">{{ _("Your response is legally required") }}</h2>
```

Comment:
if these ids are just for the tests we should probably use `data-qa="legal-response"`

Reply:
We currently have two 'id's that are the same. In basic.html and preview.html. This would cause issues as it's possible that they are both called so at least `data-qa` would negate that.

Reply:
Example of selector with data-qa rather than id:
`return '[data-qa="legal-response"]'`